### ISSUE TITLE
utoipa requires __path_ to be imported. 

### ISSUE THREAD

**Issue by Domopus** [juhaku/utoipa#1216]
Hi,

I've experienced a problem very similar to this issue: [](https://github.com/juhaku/utoipa/issues/1021)

I'm using utoipa with Rocket. 

If I do this:
`use handlers::user_handlers::get_user_by_id;

#[derive(OpenApi)]
#[openapi(
    info(description = "This is an Api to get weather information"),
    paths(get_user_by_id)
)]
struct ApiDoc;`

I have this error: `cannot find type `__path_get_user_by_id` in this scope`

As in the issue mentioned above, the current workaround is simply to import the handler this way:
`use handlers::user_handlers::{__path_get_user_by_id, get_user_by_id};`

Am I doing something wrong? Thanks a lot in advance and also for all the great job done so far :)

**Comment by juhaku** [comment]
Yes if you import the path with full path the `__path_` type need to be also imported, because utoipa expects the __path_ type to exist in same scope as the `get_user_by_id`. Alternatively you provide the full path to the handler as shown below. This way the `__path_` type does not need to be imported.
```rust
#[derive(OpenApi)]
#[openapi(
    info(description = "This is an Api to get weather information"),
    paths(handlers::user_handler::get_user_by_id)
)]
struct ApiDoc;
```

You could also define new `OpenApi` in `handlers/user_handler.rs` file like so and nest the user handlers to main `OpenApi`.
```rust
#[derive(OpenApi)]
#[openapi(
    paths(get_user_by_id)
)]
struct UserHandlers;

// in main.rs
use handlers::user_handlers::get_user_by_id;

#[derive(OpenApi)]
#[openapi(
    info(description = "This is an Api to get weather information"),
    nest(
        (path = "/", api = handlers::user_handler::UserHandlers),
//        (path = "/", api = UserHandlers), or like this but then the UserHandlers need to be imported, is suppose?
    )
)]
struct ApiDoc;
```

**Comment by Domopus** [comment]
Hi,

Thanks a lot for your answer. I actually like the solution to use openapi "nest" property. It makes the code cleaner and works like a charm 😇
